src/blocks/media-card/styles/editor.scss

```diff
@@ -1,3 +1,29 @@
+:-ms-fullscreen,
+:root .editor-styles-wrapper .wp-block .wp-block {
+	width: none;
+
```

Comment:
This was an attempted fix for the Row block width in the editor. This should changed/removed.